vue-frontend/src/components/android-app-development/CaseStudySection.vue

```diff
@@ -0,0 +1,298 @@
+        <div
+          :class="item.textalign"
+          class="tw-absolute tw-bottom-[10%] tw-flex tw-flex-col tw-text-black-core/[0.87]"
+        >
+          <span
+            class="tw-font-inter-semibold tw-text-[1.375rem] tw-leading-[1.7875rem]"
+            >{{ item.title }}</span
+          >
+          <span
+            class="tw-font-inter-regular tw-text-[1rem] tw-leading-[1.4rem]"
+            v-html="item.content"
+          ></span>
+        </div>
+      </div>
+      <div
+        v-for="item in cta"
+        :key="item"
+        class="tw-mt-7 tw-relative md:tw-hidden"
+        :class="item.className"
+      >
+        <img
+          :src="item.images[0]"
+          :srcset="`${item.images[0]} 400w, ${item.images[1]} 800w,`"
+          alt="casestudy-image"
+          class="tw-w-full tw-h-full tw-object-cover tw-mt-7"
+          loading="lazy"
+        />
+        <div
+          :class="item.textalign"
+          class="tw-absolute tw-bottom-[10%] tw-flex tw-flex-col tw-text-black-core/[0.87]"
+        >
+          <span
+            class="tw-font-inter-semibold tw-text-[1.375rem] tw-leading-[1.7875rem]"
+            >{{ item.title }}</span
+          >
+          <span
+            class="tw-font-inter-regular tw-text-[1rem] tw-leading-[1.4rem]"
+            v-html="item.content"
+          ></span>
+          <div class="tw-mt-[1rem] tw-flex tw-flex-wrap tw-justify-center">
+            <router-link
+              class="white-btn gradient-border-btn tw-flex tw-h-[2.8125rem] tw-w-max tw-items-center tw-rounded-full tw-border-none"
+              to="/contact"
+              @click.native="mixpanel.track('tap_android_development_cta4')"
+            >
+              <span
+                class="tw-font-inter-semibold tw-text-[1rem] tw-font-normal tw-leading-[1.21rem] !tw-tracking-[0] md:tw-text-[1.09375rem] md:tw-text-[1.1875rem] md:tw-leading-[1.3125rem] md:tw-leading-[1.4375rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem]"
+                >Inquire Now</span
+              >
+            </router-link>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- ------------------------ MobileUI END---------------------- -->
+    <div class="tw-hidden md:tw-block tw-bg-[#F6F6F6] tw-mt-14">
+      <div class="swiper-content tw-relative tw--mt-6 xll:tw-container">
+        <swiper
+          :slidesPerView="1"
+          :centeredSlides="true"
+          :loop="true"
+          :speed="4000"
+          :loopedSlides="50"
+          :autoplay="{
+            delay: 3000,
+            disableOnInteraction: false,
+          }"
+          :spaceBetween="20"
+          :navigation="true"
+          class="swiper-container tw-absolute tw-left-0 tw-top-[10px] tw-z-30 tw-mx-auto tw-h-full tw-w-full"
+        >
+          <swiper-slide v-for="item in cases" :key="item">
+            <div
+              class="tw-flex tw-flex-row tw-px-[4rem] tw-py-[6rem] 2xl:tw-p-[8rem] 2xll:tw-p-[11rem]"
+            >
```

Comment:
image and text part not looking good in some devices make image smaller in devices till xl and make image vertically align